oktapam/constants/descriptions/links.go

```diff
@@ -12,4 +12,5 @@ const (
 	LinkProjectGroup          = "For details, see [Add a Group to a Project](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/setup/add-a-group-to-project.htm)."
 	LinkServerEnrollmentToken = "For details, see [Enroll a server](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/setup/enrolling-a-server.htm)."
 	LinkServiceUser           = "For more information check out the [documentation](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/service-users.htm) on ASA Service Users and how they are used."
+	LinkTeamSettings          = "Add description [documentation](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/service-users.htm) on ASA Service Users and how they are used."
```

Comment:
Copy paste error.

Reply:
I just checked and I couldn't find the link to the team settings docs. I don't think we have docs for team settings. I do see API docs.

Reply:
Just link API docs for now

Reply:
That also has enough information on what exactly team settings does.